crud/src/test/resources/ITCaseCrudResourceRDBMSTest/config/config.properties

```diff
@@ -1,3 +1,4 @@
 # set default isolation strategy for tests to SEMAPHORE else breakpoints in IDE
 # when debugging unit tests will timeout with default strategy (THREAD).
 hystrix.command.default.execution.isolation.strategy=SEMAPHORE
+execution.isolation.thread.timeoutInMilliseconds=30000
```

Comment:
Yeah, I agree with you but I saw a timeout in one of the tests, which I could suppose that the Hystrix couldn't find this file in the classpath or it ignored the first line. I put this to check the second hypothesis, it didn't happen again with me or in travis, so I don't know if it worked / fixed . I can remove it but I would suggest to we have an issue to create a test and force an extreme scenario to see if have any issue (would have an special profile to run only occasionally). What do you think?